CMakeLists.txt

```diff
@@ -204,8 +205,10 @@ if(MSVC)
 
     endif()
 elseif(UNIX) #LINUX OR APPLE
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Werror")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC -Werror")
+    if (NOT IOS)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Werror")
```

Comment:
>-Werror [](start = 54, length = 7)

Need to address those before merge.

Reply:
I was able to address most of these issues. However, the iOS compile will throw a warning for an unused static function and unused variables. There are a number of those instances in the code base so I specifically turned off those two warnings for iOS builds.

---
In reply to: [139021347](https://github.com/Azure/azure-c-shared-utility/pull/109#discussion_r139021347) [](ancestors = 139021347)